### DISCUSSION TITLE
code not found

### DISCUSSION THREAD

**Discussion by abhinavreddyyy** [Cyfrin/foundry-full-course-cu#2533] [General]
I am not able to find the code for this course.
New here don't mind, so can anyone direct me as to where I can find the course.
Thank you

**Comment by EngrPips** [comment]
Hello @abhinavreddyyy, Welcome aboard. You can find the codebase for each section by clicking on the section. Here is the link https://github.com/Cyfrin/foundry-full-course-cu to get where every section of the course is listed.

**Reply by abhinavreddyyy** [reply]
Found it!
Thank you for taking out time!

**Reply by EngrPips** [reply]
You are welcome, Friend. Keep crushing it.
